config/patches/openresty/v1.11.2.1_bundled-nginx-cve-2017-7529.patch

```diff
@@ -0,0 +1,73 @@
+diff -ru openresty-1.11.2.1/bundle/nginx-1.11.2/docs/html/index.html openresty-1.11.2.4/bundle/nginx-1.11.2/docs/html/index.html
```

Comment:
I feel compelled to ask, do we have any documentation on the provenance of this patch?  Did you create it yourself based on the original upstream change?